packages/legend-application-query/src/components/data-space/DataSpaceInfo.tsx

```diff
@@ -99,7 +130,17 @@ export const QueryEditorDataspaceInfoModal = observer(
           }
           className="dataspace-info-modal"
         >
-          <ModalTitle title="About Dataspace" />
+          <div className="dataspace-info-modal__header">
+            <ModalTitle title="About Dataspace" />
+            <button
+              className="btn--dark dataspace-info-modal__header__open-btn"
+              title="Close"
+              onClick={openInTaxonomy}
+            >
+              Open in Taxonomy
```

Comment:
lets just say Open DataSpace. We may choose to show another default view here in the future.